src/pages/AddPage/AddPage.jsx

```diff
@@ -1,7 +1,46 @@
 import React from "react";
+import "./AddPage.scss";
+import { useState } from "react";
 
 const AddPage = () => {
-  return <h2>add</h2>;
+  const url = "http://localhost:3030";
+  const [text, setText] = useState("");
+  const [error, setError] = useState(null);
+  const [done, setDone] = useState(null);
+  const handleAddTodo = async () => {
+    await postTodo();
+  };
```

Comment:
This function sounds redundant. The reason is that it's only calling the `postTodo` function. You can just call `postTodo` inside the on click.

Reply:
Done.